core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.controller.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('Edit Thumbnail Modal Controller', function() {
     expect($scope.updateBackgroundColor).toHaveBeenCalled();
   });
 
-  it('should init the constiables', function() {
+  it('should evaluate $scope properties', function() {
```

Comment:
Similar to the other specs, you should be describing what the code that's being tested is doing, not what the test is doing. Ditto elsewhere.

(This one probably has something to do with "correct initialization".)

Reply:
Done.